src/components/area.js

```diff
@@ -88,7 +103,7 @@ const Area = ({ classes, size, inputProps, label, maxLength, helperText, disable
       </form>
       <div className="grid grid-cols-2">
         {helperText && <p className={areaClasses.helperText}>{helperText}</p>}
-        {!isNaN(maxLength) && <p className={areaClasses.wordCount}>{value.length || 0}/{maxLength}</p>}
+        {!isNaN(maxLength) && <p className={areaClasses.wordCount}>{value && value.length || 0}/{maxLength}</p>}
```

Comment:
would you need to consider `_value` here as well or just `value`?

Reply:
ohh I think it is better _value I will change it thanks!